xbmc/windowing/gbm/drm/DRMConnector.cpp

```diff
@@ -10,8 +10,42 @@
 
 #include "utils/log.h"
 
+#include <map>
+
 using namespace KODI::WINDOWING::GBM;
 
+namespace
+{
+
+std::map<int, std::string> connectorTypeNames = {
+    {DRM_MODE_CONNECTOR_Unknown, "unknown"},
```

Comment:
Do these type names need to be localized? Or should we also have a translatable label for each name?

Reply:
I don't think so.